### PR TITLE
Measurement Kata Feedback

### PR DESCRIPTION
**Pull request by ScottCarda-MS**
Additional feedback:
 - Most links are broken.
 - Matrix formatting under (Lesson: Computational Basis Measurements) under the question section is broken.
 - Formatting under (Lesson: Measurements in arbitrary orthogonal bases) is broken.
   - also the matrix formatting in the LaTex is broken under this section.
 - Matrix formatting under (Arbitrary basis measurements implementation) under the answer part of the question section is broken quite a bit.
 - Reference to a "problem 6" in the solution section of the final exercise.
 - I don't believe punctuation should be included when equations are not in-line, i.e. putting a period at the end of an equation.

### CODE REVIEW

File: katas/content/single_qubit_measurements/distinguish_0_and_1/solution.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
 
-In Q# the operation [`M`](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.m) can be used to measure a single qubit in the computational basis. The measurement result is a value of type `Result`: the operation `M` will return `One` if the input qubit was in the $|1\rangle$ state and `Zero` if the input qubit was in the $|0\rangle$ state. Since we need to encode the first case as `false` and the second one as `true`, we can return the result of equality comparison between measurement result and `Zero`.
+In Q# the operation [`M`](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.m) can be used to measure a single qubit in the computational basis. The measurement result is a value of type `Result` - the operation `M` will return `One` if the input qubit was in the $|1\rangle$ state and `Zero` if the input qubit was in the $|0\rangle$ state. Since we need to encode the first case as `false` and the second one as `true`, we can return the result of equality comparison between measurement result and `Zero`.
```
Comment (ScottCarda-MS) [inline katas/content/single_qubit_measurements/distinguish_0_and_1/solution.md:3]: Sometimes the file used `: ` and other times it used ` - `. I tried to consolidate around using the ` - ` unless the `:` was followed by a new-line.

File: katas/content/single_qubit_measurements/distinguish_plus_and_minus/solution.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-The input qubit is guaranteed to be either in basis state $|+\rangle$ or $|-\rangle$. This means that when measuring the qubit in the Pauli $X$ basis, the measurement will report the input state without any doubt. (Recall that these states are eigenstates for the Pauli $X$ matrix).  
+The input qubit is guaranteed to be either in basis state $|+\rangle$ or $|-\rangle$. This means that when measuring the qubit in the Pauli X basis, the measurement will report the input state without any doubt. (Recall that these states are eigenstates for the Pauli X matrix).
```
Comment (ScottCarda-MS) [inline katas/content/single_qubit_measurements/distinguish_plus_and_minus/solution.md:1]: Throughout the kata, it is just written as `Pauli X` without the math syntax. For consistency, I removed the math syntax here.

**Comment by ScottCarda-MS** [comment]
All of the in-site links don't work, but I'm wondering if that is because I'm running it off of localhost. Maybe it is a non-issue?

File: katas/content/single_qubit_measurements/distinguish_0_and_1/solution.md (hunk 1 above)
Comment (cesarzc) [reply katas/content/single_qubit_measurements/distinguish_0_and_1/solution.md:3]: Sounds like a good convention, I'll follow it for the other katas as well.

**Pull request merged**